src/main/java/duke/controller/MainWindow.java

```diff
@@ -0,0 +1,55 @@
+package duke.controller;
+
+import duke.Duke;
+
+import duke.ui.Ui;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Duke duke;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/itadori.jpeg"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/todo.jpeg"));
+
+    @FXML
+    public void initialize() {
+        scrollPane.vvalueProperty().bind(dialogContainer.heightProperty());
+        dialogContainer.getChildren().add(DialogBox.getDukeDialog(Ui.showGreeting(), dukeImage));
```

Comment:
These two lines seem a bit too long and not easy to read. Maybe you can consider separate them? 
Overall you did a really good job! Almost cannot find any coding standard violation in your project!